pkg/volume/azure_dd/azure_provision.go

```diff
@@ -43,6 +43,10 @@ type azureDiskDeleter struct {
 var _ volume.Provisioner = &azureDiskProvisioner{}
 var _ volume.Deleter = &azureDiskDeleter{}
 
+// PVCAnnotationResourceGroup is the annotation used on the PVC
+// to specify the resource group of azure managed disk that are not in the same resource group as the cluster.
+const PVCAnnotationResourceGroup = "volume.beta.kubernetes.io/resource-group"
```

Comment:
why use annotation instead of storage class parameter?

Reply:
@rootfs if we add a new parameter in azure disk storage class, this requires API change, which would be a big change, cannot cherry-picked to old version. I could send out another seperate PR to address this, and would only be in v1.12. Just let me know your requirements, thanks.

Reply:
Adding new parameters to storage class indeed is API change, however there is no strict policy around that. Nothing prevents you from backporting this into older release. You don't need to regenerate anything and you don't need sig-apimachinery approval. Just update the code + docs.